store/views.py

```diff
@@ -125,18 +126,25 @@ class LoginView(TemplateView):
 
     def get(self, request):
         if request.user.is_authenticated:
-            return redirect("/")
+           if request.user.user_type == 'seller':
+                return redirect("dashboard")  # Redirect sellers to the dashboard
+           else:
+                return redirect("/")  # Redirect buyers to the home page
```

Comment:
Please explain why did you use redirect("/") here?

Reply:
i had two types of users, buyers and sellers and i wanted to redirect buyers to the homepage after they logged in and sellers to the dashboard, hence i used the if else condition to check if the user is seller or buyer and redirect them accordingly

Reply:
instead of / use name of url.

Reply:
noted sir